package.json

```diff
@@ -8,7 +8,7 @@
     "classnames": "2.1.2",
     "flux": "^2.0.3",
     "json-stable-stringify": "1.0.0",
-    "keen-analysis": "^1.1.0",
+    "keen-analysis": "1.1.0",
```

Comment:
Thanks for the heads up about the keen-analysis version. This just in: looks like it's being addressed in https://github.com/keen/explorer/pull/197.